test/libgit2.jl

```diff
@@ -794,6 +799,5 @@ mktempdir() do dir
             end
         end
     =#
+    end
```

Comment:
I see what happened. You want to end the now-commented-out `@testset "SSH" begin also inside the block comment